torchdet3d/utils/ie_wrappers.py

```diff
@@ -159,3 +161,311 @@ def crop(frame, rect):
         x0, y0, x1, y1 = rect
         crop = frame[y0:y1, x0:x1]
         return crop
+
+
+class Track:
```

Comment:
Tracker is not related to IE wrappers, so I'd prefer have it in another file.

Reply:
moved all related stuff to tracking_tools.py